src/Tracing/Instrumentation/EventSubscriber/CommandEventSubscriber.php

```diff
@@ -46,7 +46,10 @@ public function onCommand(ConsoleCommandEvent $event): void
     {
         $name = $event->getCommand()?->getName() ?: 'unknown-command';
 
-        $this->span = $this->startSpan(sprintf('cli %s', $name), ['command' => $name]);
+        $startTime = $_SERVER['REQUEST_TIME_FLOAT'] ?? microtime(true); // Float with microsecond precision
+        $kernelBootTime = round(microtime(true) - $startTime, 3);
```

Comment:
I think `REQUEST_TIME_FLOAT` is always present, no matter what.
I would also suggest to move that has first instruction, if microseconds matters lets be as precise as possible.
```suggestion
        $startTimeInMs = $_SERVER['REQUEST_TIME_FLOAT'];
        $kernelBootTime = round(microtime(true) - $startTimeInMs, 3);
```

I don't find how to add a comment on a line that you did not modify so I put it here:
You must change the priority of the event, a lot of listeners (sentry, monolog, debug, dump, etc) have a higher priority. Otherwise it's not really the "kernel boot time".

Reply:
@camilledejoye  Are you saying that `REQUEST_TIME_FLOAT` is set in CLI sapi context?